frontend/src/lib/UriUtil.ts

```diff
@@ -115,3 +115,23 @@ export function buildMediaUri(uri: string): string {
     ? buildHttpUri(getWindowBaseUriParts(), uri)
     : uri
 }
+
+/**
+ * Check if the given url follows the url pattern which could include
+ * a wildcard.
+ */
+export function isValidURL(pattern: string, hostname: string): boolean {
+  const splittedPattern = pattern.split(".")
```

Comment:
nit: I think it should be `splitPattern` as `split` remains `split` in the past tense